src/components/common/ColonyTotalFunds/ColonyTotalFundsSelectedToken.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useMemo, useState, useEffect } from 'react';
+import { defineMessages } from 'react-intl';
+
+import Icon from '~shared/Icon';
+import Numeral from '~shared/Numeral';
+// import { MiniSpinnerLoader } from '~shared/Preloaders';
+// import { Colony, useTokenBalancesForDomainsQuery } from '~data/index';
+import { Address } from '~types/index';
+import { getTokenDecimalsWithFallback } from '~utils/tokens';
+// import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
+import { useColonyContext } from '~hooks';
+import { notNull } from '~utils/arrays';
+
+import ColonyTotalFundsPopover from './ColonyTotalFundsPopover';
+
+import styles from './ColonyTotalFunds.css';
+import IconTooltip from '~shared/IconTooltip';
+
+const MSG = defineMessages({
+  loadingData: {
+    id: 'dashboard.ColonyTotalFundsSelectedToken.loadingData',
+    defaultMessage: 'Loading token information...',
+  },
+  tokenSelect: {
+    id: 'dashboard.ColonyTotalFundsSelectedToken.tokenSelect',
+    defaultMessage: 'Select Tokens',
+  },
+});
+
+type Props = {
+  children?: React.ReactChild;
+};
+
+const displayName = 'dashboard.ColonyTotalFundsSelectedToken';
+
+const ColonyTotalFundsSelectedToken = ({ children }: Props) => {
```

Comment:
I think it would be fine to call this `SelectedToken`. It's in the `ColonyTotalFunds` directory so don't think we need to repeat the directory name in the component name